dockstore-webservice/src/main/java/io/dockstore/webservice/core/metrics/MetricsByStatus.java

```diff
@@ -69,6 +69,12 @@ public class MetricsByStatus {
     @Schema(description = "Aggregated cost metrics in USD for the status")
     private CostStatisticMetric cost;
 
+    @Valid
+    @OneToOne(cascade = CascadeType.ALL, orphanRemoval = true)
+    @JoinColumn(name = "dailyexecutioncountsid", referencedColumnName = "id")
+    @Schema(description = "daily execution count time series for the status")
```

Comment:
Easy to change, I wonder if we get enough traffic for daily to be useful, maybe we should start with weekly or monthly? 
This is just a prototype though

Reply:
> Easy to change, I wonder if we get enough traffic for daily to be useful, maybe we should start with weekly or monthly? This is just a prototype though

Sure, might change it to weekly.  Should be a quick change, so let's see what daily looks like first.